.cci.jenkinsfile

```diff
@@ -35,6 +35,26 @@ coreos.pod([image: 'registry.fedoraproject.org/fedora:30', privileged: true, kvm
               cosa_cmd("buildupload --dry-run s3 --acl=public-read my-nonexistent-bucket/my/prefix")
           }
       }
+
+      stage("Pruning test") {
+        coreos.shwrap("cat /srv/builds/builds.json")
+        cosa_cmd("build ostree --force-image")
+        cosa_cmd("build ostree --force-image")
+        cosa_cmd("build ostree --force-image")
+        coreos.shwrap("cat /srv/builds/builds.json")
```

Comment:
There is `jq` in the container. It should be pretty easy to sanity-check the condition after each build. E.g.:

```
jq -e ".builds|length == 3" builds/builds.json
```

Reply:
Updated, PTAL